.travis.yml

```diff
@@ -2,9 +2,6 @@ language: node_js
 node_js:
   - 4
   - 6
-cache:
-  directories:
-    - node_modules
```

Comment:
This was removed temporarily. Till this issue is not fixed https://github.com/rstacruz/pnpm/issues/315